frontend/public/components/dashboard/dashboards-page/cluster-dashboard/status-card.tsx

```diff
@@ -139,8 +142,8 @@ const ClusterAlerts = connectToFlags(FLAGS.OPENSHIFT)(
       return (
         <AlertsBody
           isLoading={
-            flagPending(isOpenshift) ||
-            (isOpenshift ? !(alertsResponse && cvLoaded) : !alertsResponse)
+            flagPending(hasCVResources) ||
+            (hasCVResources ? !(alertsResponse && (cvLoaded || !!cvError)) : !alertsResponse)
```

Comment:
I think this could be simpler if I haven't gotten the logic wrong:

```suggestion
            !alertsResponse || (isOpenShift && !cvLoaded && !cvError)
```

Reply:
you got it right, fixed